pkg/gardenadm/botanist/extensions.go

```diff
@@ -53,3 +60,28 @@ func ComputeExtensions(seedName string, controllerRegistrations []*gardencorev1b
 
 	return extensions, nil
 }
+
+// ReconcileExtensionControllerDeployments reconciles the extension controller deployments.
+func (b *AutonomousBotanist) ReconcileExtensionControllerDeployments(ctx context.Context, networkAvailable bool) error {
```

Comment:
Might be nitpicking, but this rather reconciles the `ControllerInstallations` and not the `ControllerDeployments`. Hence, the naming is slightly off.